validator/schemas/project.py

```diff
@@ -0,0 +1,88 @@
+# pylint: disable=duplicate-code
+"""The project schema managed by Nestor"""
+
+PROJECT_SCHEMA = {
+    "type": "object",
+    "properties": {
+        "env": {},
+        "domain": {},
+        "env_suffix": {},
+        "domain_prefix": {},
+        "docker": {
+            "type": "object",
+            "properties": {
+                "build": {
+                    "type": "object",
+                    "properties": {
+                        "variables": {
+                            "type": "object",
+                            "patternProperties": {
+                                "": {"type": "string", "pattern": "^([A-Z_]*)",},
+                            },
+                        },
+                    },
+                },
+                "registries": {
+                    "type": "object",
+                    "properties": {
+                        "docker.com": {
+                            "type": "array",
+                            "items": {
+                                "type": "object",
+                                "properties": {
+                                    "id": {"type": "string",},
+                                    "organization": {"type": "string",},
+                                },
+                            },
+                        },
+                    },
+                },
+            },
+            "required": ["build", "registries"],
+            "additionalProperties": False,
+        },
+        # TODO to be defined pylint: disable=fixme
```

Comment:
FYI we used "Awaiting for implementation" to go around this warning in [some other files](https://github.com/ChauffeurPrive/nestor-api/blob/master/nestor_api/lib/app.py#L7) 😅